etl/src/main/java/org/pih/hivmigration/etl/sql/Migrator.java

```diff
@@ -1,27 +1,81 @@
 package org.pih.hivmigration.etl.sql;
 
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
 import org.apache.commons.lang3.time.StopWatch;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
+import javax.sound.midi.SysexMessage;
+import java.lang.reflect.InvocationTargetException;
+
 /**
  * Main class for executing the HIV migration using Spark
  */
+@Parameters(separators = "=")
 public class Migrator {
 
     private static final Log log = LogFactory.getLog(Migrator.class);
 
+    @Parameter(names={"--step", "-s"}, description="Only run a single step. Should be the prefix of a Migrator, like 'User'.")
+    private String step;
+
+    @Parameter(names={"--revert", "-r"}, description="Revert previous changes before the corresponding step.")
+    private boolean shouldRevert = false;
+
+    @Parameter(names={"--limit", "-l"}, description="Limit the number of rows to import.")
+    private int limit = -1;
+
     /**
      * Run the application
      */
 	public static void main(String[] args) {
         log.info("Starting up HIV Migrator");
-        revert(new UserMigrator());
-        migrate(new UserMigrator());
+        Migrator migrator = new Migrator();
+        JCommander.newBuilder()
+                .addObject(migrator)
+                .build()
+                .parse(args);
+        migrator.main();
+    }
+
+    public void main() {
+	    if (step != null && !step.isEmpty()) {
+	        log.info("Only running " + step + "Migrator.");
+        }
+	    if (shouldRevert) {
+	        log.info("Will revert before each step.");
```

Comment:
Nit, but based on what we were talking about re: revert order, this should probably just read "Will revert before running".